docker/app/Dockerfile

```diff
@@ -18,3 +18,6 @@ COPY BOOT-INF/classes /app
 
 ENV spring_profiles_active docker
 CMD ["ash", "-c", "java -cp app:app/lib/* io.tolgee.Application"]
+
+HEALTHCHECK --interval=10s --timeout=3s --retries=20 \
+    CMD [[ "$(wget -q -O -  http://127.0.0.1:8080/actuator/health)" == *"UP"* ]] || exit 1
```

Comment:
@JanCizmar yes it is. If it is fine for you that I use an env explicit for the healthcheck?

```suggestion

ENV HEALTHCHECK_PORT=8080

HEALTHCHECK --interval=10s --timeout=3s --retries=20 \
    CMD [[ "$(wget -q -O -  http://127.0.0.1:$HEALTHCHECK_PORT/actuator/health)" == *"UP"* ]] || exit 1
```

Reply:
Yep! That works!